src/tigr_linux.c

```diff
@@ -511,6 +514,21 @@ static void tigrProcessInput(TigrInternal* win, int winWidth, int winHeight) {
         prevButtons = buttons;
     }
 
+    // Retrieve mouse wheel events - Cannot be done in the XQueryPointer call
+    // -> https://comp.unix.programmer.narkive.com/eOnjkQ6L/xlib-xquerypointer-and-button4mask-button5mask#post2
+    // Button4 = WheelUp / Button5 = WheelDown
+    XEvent mouseButtonEvent;
+	if (XPending(win->dpy)) {
```

Comment:
Since we are not processing events in a loop here, we will get at most one wheel event per `tigrUpdate` call. This should lead to events getting queued up / delayed.